data_hub_api/docmaps/v2/codecs/elife_manuscript.py

```diff
@@ -101,6 +101,11 @@ def get_docmap_elife_manuscript_output_for_vor(
     return {
         'type': 'version-of-record',
         'doi': manuscript_version_doi,
+        'published': (
+            manuscript_version['vor_publication_date'].strftime('%Y-%m-%d')
```

Comment:
minor: we could also use `.isoformat()` as we do in some other cases?

BTW how is this currently working, isn't `vor_publication_date` coming from BigQuery currently a string?

Reply:
it is not string it is date now maybe haven’t been updated yet in prod I have recently merged a PR to fix this